lib/src/serializers/json_api.dart

```diff
@@ -74,6 +74,9 @@ class JsonApiDocument {
   Map<String, dynamic> relationships;
   List<dynamic> included;
   List<dynamic> errors;
+  Map<String, List<String>> _idsCache = Map<String, List<String>>();
+  Map<String, List<JsonApiDocument>> _docsCache =
```

Comment:
@halil81 This is unused, as reported by Dart analyzer

Reply:
@algodave I removed _docsCache. Thanks!